### DISCUSSION TITLE
Filesystem media tags

### DISCUSSION THREAD

**Discussion by epursimuove70** [music-assistant/support#2767] [Feature requests and ideas]
It would be nice to have the ID3 tag information available with the HA Search or a separate Media ID service call.  Actually it started with "I wish I could see the ID3 info" in MA when you drill down on the Show Info for the track.

**Comment by OzGav** [comment]
A lot of information (if not all the tag info - I haven't done a line by line comparison) can now be retrieved via the `mass.get_library` service call.
